var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -16,14 +16,21 @@ class Petsc(Package):
     """
 
     homepage = "http://www.mcs.anl.gov/petsc/index.html"
-    url      = "http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-3.5.3.tar.gz"
-    git      = "https://gitlab.com/petsc/petsc.git"
-
+    git = "https://gitlab.com/petsc/petsc.git"
     maintainers = ['balay', 'barrysmith', 'jedbrown']
 
+    def url_for_version(self, version):
+        if version >= Version('3.13.0'):
+            # petsc-lite tarballs are smaller by skipping docs
+            return "http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-lite-{}.tar.gz".format(version)
```

Comment:
```suggestion
            return "http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-lite-{0}.tar.gz".format(version)
```
This is needed for Python 2.6 support

Reply:
updated

Reply:
I was hesitant to change all the currently listed version from petsc-v.tar.gz to petsc-lite-v.tar.gz - hence this extra complexity